models/reviews.js

```diff
@@ -27,3 +27,24 @@ exports.updateReview = (review_id, inc_votes) => {
       return rows[0];
     });
 };
+
+exports.selectReviews = (category) => {
+  const queryValues = [];
+  let queryStr = `SELECT reviews.*, COUNT(comment_id) AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id`;
+  if (category) {
+    if (!["social deduction", "dexterity", "euro game", "children's games"].includes(category)) {
+      return Promise.reject({ status: 400, msg: "Invalid category query" });
```

Comment:
Nice - have a think about how we could make this dynamic using sql queries to the db to validate the category options available... something to think about going forwards, wouldn't say it's mandatory right here but may be an idea if you're looking to add improvements at a later date!